tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -455,6 +455,21 @@ def test_shared_gather(mode):
             assert op.get_element_type() == weight_name_vs_type[op_name]
 
 
+@pytest.mark.parametrize("all_layers", (True, False))
+def test_shared_gather_all_layers(all_layers):
```

Comment:
Did you manage to catch the incorrect behavior with the old code? Seems like this test passed before

Reply:
Looks like the old behavior was also correct. I couldn't find incorrect configuration.